types/result.go

```diff
@@ -107,13 +107,15 @@ func newTxResponseCheckTx(res *ctypes.ResultBroadcastTxCommit) *TxResponse {
 		txHash = res.Hash.String()
 	}
 
+	parsedLogs, _ := ParseABCILogs(res.CheckTx.Log)
+
 	return &TxResponse{
 		Height:    res.Height,
 		TxHash:    txHash,
 		Codespace: res.CheckTx.Codespace,
 		Code:      res.CheckTx.Code,
 		Data:      strings.ToUpper(hex.EncodeToString(res.CheckTx.Data)),
-		RawLog:    res.CheckTx.Log,
+		Logs:      parsedLogs,
```

Comment:
is it possible to add a new field rather than replacing it, for backwards compatibility (in case anyone uses raw log)

Reply:
we added the rawLog in the last patch so it shouldn't have been used by anybody (was used for logging purposes), but fine to leave both there